lib/native/binding.js

```diff
@@ -7,6 +7,33 @@ import * as path from 'path';
 import { platform, arch } from 'os';
 import { existsSync } from 'fs';
 import { versions } from 'process';
+import child_process from "child_process";
+
+function getCRuntime() {
+    let non_linux_runtime_tag = 'cruntime';
+    let musl_tag = 'musl';
```

Comment:
Maybe make these enumerated constants (ala https://github.com/aws/aws-iot-device-sdk-js-v2/blob/main/lib/greengrasscoreipc/model.ts#L19-L31) rather than repeating the string literal?

Reply:
Thanks, fixed.